java/client/src/main/java/glide/connectors/handlers/PubSubMessageQueue.java

```diff
@@ -0,0 +1,78 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.connectors.handlers;
```

Comment:
Is it good place for this class?
Probably, should be moved together with `MessageHandler` to `glide.managers`

Reply:
I think it's OK here. It's tightly coupled to MessageHandler (could even be an inner class of it).